src/track/cueinfo.h

```diff
@@ -21,6 +21,14 @@ enum class CueType {
                       // sound; not shown to user
 };
 
+enum class CueFlag {
+    None = 0,
+    /// Currently only used when importing locked loops from Serato Metadata.
+    Locked = 1,
+};
```

Comment:
Please add a comment about the implication when changing or adding flags.

Reply:
What implications? There are none. These flags are not stored in the DB. We only have the "locked" flag for test cases at the moment.

Reply:
I mean to add a comment like:
``` 
// Don't change the numbers, they are stored in the database.  
```
or more if there is more to consider.

Reply:
But they are not. The flags are discarded before importing them into the DB.

Reply:
I'll try to explain it a bit more:

All Serato Cues are imported like this:

Binary Serato Metadata -> Parsed Serato Metadata Objects -> CueInfo Objects -> Database

The flags are discarded before database import because we don't support them in Mixxx.

So why even have such a member in CueInfo?
Because then we can have roundtrip tests that check the conversion:

Parsed Serato Metadata Objects -> CueInfo Objects ->   Parsed Serato Metadata Objects

We might want to add support for a "locked" flag in Mixxx, too. But that is out of scope for this PR.

Reply:
Ah OK, so the comment may look like: 

// These flags are only used internally.